controllers/userController.js

```diff
@@ -101,6 +102,41 @@ router.post('/:id/password', verifyToken, (req, res) => {
 	});
 });
 
+// Changes a don's information (only accessible by dons)
+router.post('/:id/doninfo', verifyToken, (req, res) => {
+	if(req.user.role&1 != 1) {
+		console.error("User "+ req.user._id +"is requesting don info when user is not a don");
+		//requestor is not a don
+		res.status(403);
+	}
+	User.findOne({_id: req.params.id}).then(user => {
+		if(user.role&1 != 1){
+			//user is not a don
+			res.status(403);
+		}else{
+			user.donInfo = req.body
+			user.update(user);
+			//set timer to turn off don automagically
+			if(user.donInfo.isOn){
+				setTimer(user.donInfo.clockOut, ''+user._id, {},() =>{
+					User.findOne({_id: user._id}).then(user => {
+						console.log("Clock out don " + user.username);
```

Comment:
A couple of these `console.log` statements could be taken out as they're mainly for debug purposes I'm guessing, but it doesn't matter too much.

Reply:
Ideally we should probably agree on/set up a linter...